llama_finetuning.py

```diff
@@ -127,14 +147,29 @@ def main(**kwargs):
 
         mixed_precision_policy, wrapping_policy = get_policies(fsdp_config, rank)
         my_auto_wrapping_policy = fsdp_auto_wrap_policy(model, LlamaDecoderLayer)
-   
+
+        # given the fast evolving PRs around meta device init, I am not sure
+        # what is the best param_init_fn atm, maybe we can switch to simple to_emtpy.
```

Comment:
I think `to_empty(recurse=False)` seems okay.

Reply:
@awgu thank you! I will make the change.

Reply:
addressed https://github.com/facebookresearch/llama-recipes/pull/77/commits/1e64fc98d9296b28b7d34b11c28498f50af69e8b